packages/nimble-components/src/table-column/date-text/tests/table-column-date-text.spec.ts

```diff
@@ -26,187 +26,558 @@ async function setup(): Promise<Fixture<Table<SimpleTableRecord>>> {
+        html`<nimble-table style="width: 700px">
+                <${tableColumnDateTextTag} field-name="field" group-index="0"
+                    format="custom"
+                    custom-locale-matcher="lookup"
+                    custom-weekday="short"
+                    custom-era="short"
+                    custom-year="2-digit"
+                    custom-month="short"
+                    custom-day="2-digit"
+                    custom-hour="2-digit"
+                    custom-minute="2-digit"
+                    custom-second="2-digit"
+                    custom-time-zone-name="longOffset"
+                    custom-format-matcher="basic"
+                    custom-hour12="false"
+                    custom-time-zone="America/Chicago"
+                    custom-calendar="hebrew"
+                    custom-day-period="short"
+                    custom-numbering-system="fullwide"
+                    custom-date-style="long"
+                    custom-time-style="long"
+                    custom-hour-cycle="h23"
+                    >
+                    Column 1
+                </${tableColumnDateTextTag}>
+                <${tableColumnDateTextTag} field-name="anotherField">
+                    Squeeze Column 1
+                </${tableColumnDateTextTag}>
+            </nimble-table>`
+    );
+}
+
 describe('TableColumnDateText', () => {
     let element: Table<SimpleTableRecord>;
     let connect: () => Promise<void>;
     let disconnect: () => Promise<void>;
     let tablePageObject: TablePageObject<SimpleTableRecord>;
     let pageObject: TableColumnDateTextPageObject<SimpleTableRecord>;
+    let column: TableColumnDateText;
 
-    beforeEach(async () => {
-        ({ element, connect, disconnect } = await setup());
-        tablePageObject = new TablePageObject<SimpleTableRecord>(element);
-        pageObject = new TableColumnDateTextPageObject(tablePageObject);
-    });
-
-    afterEach(async () => {
-        await disconnect();
-    });
+    describe('no static config', () => {
+        beforeEach(async () => {
+            ({ element, connect, disconnect } = await setup());
+            tablePageObject = new TablePageObject<SimpleTableRecord>(element);
+            pageObject = new TableColumnDateTextPageObject(tablePageObject);
+            await connect();
+            await waitForUpdatesAsync();
+            column = element.columns[0] as TableColumnDateText;
+        });
 
-    it('reports column configuration valid', async () => {
-        await connect();
-        await waitForUpdatesAsync();
+        afterEach(async () => {
+            await disconnect();
+        });
 
-        const firstColumn = element.columns[0] as TableColumnDateText;
+        it('reports column configuration valid', () => {
+            expect(column.checkValidity()).toBeTrue();
+        });
 
-        expect(firstColumn.checkValidity()).toBeTrue();
-    });
+        const badValueData = [
```

Comment:
@rajsite what's the outlook on #1379 being completed so we could use the new templatized pattern here? If it still needs to bake longer do you have an opinion on whether it'd be better to at least use the soon-to-be-deprecated `getSpecTypeByNamedList`?

Reply:
Yep parameterized tests should use whatever the current pattern is, which is currently getSpecTypeByNamedList. Don't think it's worth blocking PRs on the new pattern. After the new pattern is merged tests should prefer it.